### PR TITLE
Improve charset handling

### PR DESCRIPTION
**Pull request by Godnite**
## Description
Handle mbstring charsets support

## Changelist
See README.md

## Issue
None

### CODE REVIEW

**Comment by Godnite** [comment]
Some method names in tests aren't correct. They should be corrected using camel case rule.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [![Codecov](https://img.shields.io/codecov/c/github/rancoud/security?logo=codecov)](https://codecov.io/gh/rancoud/security)
 [![composer.lock](https://poser.pugx.org/rancoud/security/composerlock)](https://packagist.org/packages/rancoud/security)
 
-Security.  
+Security.
```
Comment (rancoud) [inline README.md:11]: We can description from issue #3 
`Escape string to output HTML (and JS).`

**Comment by rancoud** [comment]
Use `static::` intead of `self::` because self is stuck to the main class, static will folow any extends.
See https://www.php.net/manual/fr/language.oop5.late-static-bindings.php

File: README.md (hunk 1 above)
Comment (rancoud) [inline README.md:11]:
```suggestion
Escape string to output HTML (and JS).
```

**Comment by Godnite** [comment]
> 
> 
> Use `static::` instead of `self::` because self is stuck to the main class, static will folow any extends.
> See https://www.php.net/manual/fr/language.oop5.late-static-bindings.php

I know you commonly use static:: but the methods that I call using self:: handle all supported charsets of PHP (more precisely html_* functions) so it can't handle more charset. That why, I think that those methods shouldn't be overridden and I want to call them only using self::

If you disagree, feel free to change that

**Comment by rancoud** [comment]
I'm taking the next PR because it's more advanced

**Pull request closed without merging**